base/setup/lib/utils/arcname.c

```diff
@@ -877,7 +900,14 @@ ArcPathToNtPath(
             return FALSE;
         }
     }
-    NtPath->Length = wcslen(NtPath->Buffer) * sizeof(WCHAR);
+
+    PathLength =  wcslen(NtPath->Buffer);
```

Comment:
Remove extra space between '=' and 'wcslen'.

Reply:
Done